packages/styled-icons/README.md

```diff
@@ -158,6 +199,22 @@ export const RedLock = styled(Lock)`
 `
 ```
 
+### Base Icon Styles
+
+If you wish to style all icons at once, you can create a wrapper styled component that imparts a particular style to all icons contained within the wrapper by targeting the `StyledIconBase` component:
+
+```javascript
+import styled from 'styled-components'
+import {StyledIconBase} from '@styled-icons/styled-icon'
+
+export const IconStyleWrapper = styled.div`
+  ${StyledIconBase} {
```

Comment:
I believe this syntax also requires the babel-plugin to handle component selectors. 

We might be able to get rid of it by doing something similar to what I suggested for customizing the `styled` tag. 

```js
StyledIconBase.mixin = (props)  => css``
```

Then have StyledIconBase inject that if it exists

Reply:
I believe this doesn't require the babel plugin actually - I did some digging and it looks like component CSS classnames don't depend on the plugin, though I may have missed something.  Trying this out on CodeSandbox seems to support this, where it works without the plugin present.

Reply:
You're correct. I was thinking of emotion which requires the babel plugin to do so. https://emotion.sh/docs/styled#targeting-another-emotion-component